### ISSUE TITLE
the funni

### ISSUE THREAD

**Issue by Gentleman2292** [Errorcrafter/tidehack-plus-leaked#2]
https://github.com/TldeHack-Dev/tidehackplus-runtime-downloads/blob/main/tidetweaker-ch.exe?raw=true

**Comment by Errorcrafter** [comment]
yeah im aware it exists, but its obfuscated or i'm decompiling it wrong or i'm retarded

**Comment by ghost** [comment]
ITS A PYTHON COMPILED EXE LOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOL

Edit 2: this is the funi stuff it grabs  
![acaiberii imagery - ajdXI.png](http://img.acaiberii.repl.co/images/png/ajdXI.png)

Edit 3: it's made with PyInstaller and I just got the pycs

Edit 4: i'm trying to decompile it the bytecode is for python 3.9 so i have to install pycdc for it

**Comment by Errorcrafter** [comment]
WAIT FOR REAL

**Comment by ghost** [comment]
I can't find any discord strings in the extremely obfuscated python bytecode.... I'll have to wait until uncompyle6 upgrades to 3.9... the main dev isnt interested in continuing the project.

**Comment by ghost** [comment]
This is all the stuff I could get no discord webhook link to do the funi to

![acaiberii imagery - zfDf6.png](http://img.acaiberii.repl.co/images/png/zfDf6.png)

As you can see, AllahCorp powered AntiDecompile(TM) (actually pjorion) technologies are in use here.


Also yeah I probably got logged by this while I decompiled cuz im using extremely crappy decompilers fml

Also I just found a pjorion deobfuscator and I already deleted the disassembled exe LOOOOOOOL I AM SO DUMB XDDD

Edit #69420: bruh that deobfuscator is extremely outdated

**Comment by Errorcrafter** [comment]
that image link is dead bruh

**Comment by Gentleman2292** [comment]
funni shits

maybe install dotpeek
it decompiles .exes

**Comment by Gentleman2292** [comment]
https://www.jetbrains.com/decompiler/

**Comment by Gentleman2292** [comment]
nvm its for c# and similar langs

**Comment by Gentleman2292** [comment]
h
https://github.com/extremecoders-re/PjOrion-Deobfuscator

**Comment by ghost** [comment]
> h https://github.com/extremecoders-re/PjOrion-Deobfuscator

I was using that deobfuscator but it's for old Python and PjOrion and hasn't been updated since July 2017. We officially have no way of getting the source code!!!

**Comment by ghost** [comment]
also replit is very funny and keeps deleting new files lol
